### DISCUSSION TITLE
harvester continuosly connect / reconnect

### DISCUSSION THREAD

**Discussion by Dredok** [Chia-Network/chia-blockchain#5694] [Support]
Hi,
I have two full nodes, one running on latest github branch and other running on docker version.
The non-docker fullnode is running harvester and farmer; there are also two other computers running harvester each of them connected to this full node. These two harvesters connect and disconnect continuosly to the farmer; no matter what I can, they connect and disconnect ... They get messages to look for proofs from time to time but i'm worried of this connection/disconnection infinite loop.

However, the other full node running on docker version does not have this issue. It has two harvesters connected and they connect and keep connected, without disconnecting continuosly.

All machines run on linux, I'm a kind of tech savvy so I can test "advanced" stuff if you want.

**Comment by BrandtH22** [comment]
@Dredok , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
